core/src/main/java/org/frankframework/jdbc/XmlQuerySender.java

```diff
@@ -421,34 +385,6 @@ private Message executeUpdate(Connection connection, String tableName, String qu
 		}
 	}
 
-	private boolean existBlob(List<Column> columns) {
-		for (Column column : columns) {
-			if (column.getType().equalsIgnoreCase(TYPE_BLOB)) {
-				return true;
-			}
-		}
-		return false;
-	}
-
-	private boolean existClob(List<Column> columns) {
-		for (Column column : columns) {
-			if (column.getType().equalsIgnoreCase(TYPE_CLOB)) {
-				return true;
-			}
-		}
-		return false;
-	}
-
-	private int countParameters(List<Column> columns) {
-		int parameterCount = 0;
-		for (Column column : columns) {
-			if (column.getParameter() != null) {
-				parameterCount++;
-			}
-		}
-		return parameterCount;
-	}
-
 	private Message alterQuery(Connection connection, String sequenceName, int startWith) throws SenderException {
 		String callQuery = "declare" + " pragma autonomous_transaction;" + " ln_increment number;" + " ln_curr_val number;" + " ln_reset_increment number;" + " ln_reset_val number;" + "begin" + " select increment_by into ln_increment from user_sequences where sequence_name = '" + sequenceName + "';" + " select " + (startWith - 2) + " - " + sequenceName + ".nextval into ln_reset_increment from dual;" + " select " + sequenceName + ".nextval into ln_curr_val from dual;" + " EXECUTE IMMEDIATE 'alter sequence " + sequenceName + " increment by '|| ln_reset_increment ||' minvalue 0';" + " select " + sequenceName + ".nextval into ln_reset_val from dual;" + " EXECUTE IMMEDIATE 'alter sequence " + sequenceName + " increment by '|| ln_increment;" + "end;";
```

Comment:
You didn't change it, but I am just a bit surprised on the way how this is written, with many separate Strings.